source/financial-aid.rst

```diff
@@ -68,8 +68,61 @@ It may be sensible to say that anyone who applies for financial aid, regardless
 Process
 -------
 
+Data Collection
+~~~~~~~~~~~~~~~
+
 I will be coordinating with George to formalise the data collection required for financial aid. In particular, we’ll want to set up some forms that ask the questions we believe need to be asked.
 
+Proposed Questions
+^^^^^^^^^^^^^^^^^^
+
+The proposed questions are below, including the type of answer that can be provided, as will the purpose of the question.
+
++----------------------------+----------------+---------------------------------------------+
+| Question                   | Answer Type    | Purpose                                     |
++============================+================+=============================================+
+| Legal Name                 | Free text      | For legal reasons, this is data we need.    |
```

Comment:
If we're going to need it, we can ask for it later.  Feels potentially a little intimidating to me.